cli/cli/commands/import/import.go

```diff
@@ -241,6 +251,36 @@ func convertComposeProjectToStarlark(compose *types.Project) (string, map[string
 	return script, requiredFileUploads, nil
 }
 
+func getMemoryMegabytesReservation(deployConfig *types.DeployConfig) int {
+	if deployConfig == nil {
+		return 0
+	}
+	reservation := 0
+	if deployConfig.Resources.Reservations != nil {
+		reservation = int(deployConfig.Resources.Reservations.MemoryBytes) / bytesToMegabytes
+		logrus.Debugf("Converted '%v' bytes to '%v' megabytes", deployConfig.Resources.Reservations.MemoryBytes, reservation)
+	}
+	return reservation
+}
+
+func getMilliCpusReservation(deployConfig *types.DeployConfig) int {
+	if deployConfig == nil {
+		return 0
+	}
+	reservation := 0
+	if deployConfig.Resources.Reservations != nil {
```

Comment:
similar is resources bound to be set?
if resources is set is nanocpus definitely set? (probably as its a float?)